albo.py

```diff
@@ -73,9 +80,13 @@ def send_telegram_attachments(pdf_links: List[str]) -> None:
     if header in break_line_headers:
       message += "\n"
     message += "*" + header + "*: " + escape_char(row[i].span.string) + "\n"
-  send_telegram_message(message)
-  attachments = ["http://albo.unict.it/" + list_item['href'] for list_item in tr.find_all('a')]
-  send_telegram_attachments(attachments)
+  try:
+    send_telegram_message(message)
+    attachments = ["http://albo.unict.it/" + list_item['href'] for list_item in tr.find_all('a')]
+    send_telegram_attachments(attachments)
+  except ValueError as err:
+    error_string = "response sent: " + err.args[0] + "\n" + "last parameter: "+ err.args[1]
+    print (error_string) #Insert here bot sending error message in group
```

Comment:
you can create a CHAT_ID_DEV and use this
```suggestion
    send_telegram_message(error_string, CHAT_ID_DEV)
```